test/src/org/labkey/test/tests/idri/FormulationsTest.java

```diff
@@ -319,16 +320,14 @@ protected void defineParticleSizeAssay()
         ReactAssayDesignerPage assayDesignerPage = _assayHelper.createAssayDesign("Particle Size", PS_ASSAY)
             .setDescription(PS_ASSAY_DESC);
 
-        assayDesignerPage.goToBatchFields();
-        assertElementPresent(Locator.tagWithClass("div", "domain-form-no-field-panel"));
+        DomainFormPanel batchPanel = assayDesignerPage.goToBatchFields();
+        assertEquals("Batch fields", Collections.emptyList(), batchPanel.fieldNames());
 
         DomainFormPanel runPanel = assayDesignerPage.goToRunFields();
-        assertThat("Missing expected run property", runPanel.getField("IDRIBatchNumber"), CoreMatchers.notNullValue());
+        assertThat("Run fields", runPanel.fieldNames(), hasItems("IDRIBatchNumber"));
 
         DomainFormPanel resultPanel = assayDesignerPage.goToFieldProperties("Result Properties");
```

Comment:
```suggestion
         DomainFormPanel resultPanel = assayDesignerPage.goToResultFields();
```

Reply:
@labkey-tchad 
We had it as using goToResultFields() for some other tests in the previous fb, but there is a subtle difference. The goToResultFields(), which was added for GPAT assay scenarios, uses "Results Properties" and some of these assays have unfortunately decided to drop the "s" and call their third domain "Result Properties". That is why we needed to use the goToFieldProperties() helper instead.